### PR TITLE
Fix lucene,activemq and owa data paths

### PR DESCRIPTION
**Pull request by enyachoke**

### CODE REVIEW

File: src/bahmni-helm/templates/apps/openmrs/openmrs-deployment.yml [hunk 1]
```diff
@@ -86,7 +86,16 @@ spec:
...
+            - name: {{ .Release.Name }}-data
+              subPath: "openmrs/lucene"
+              mountPath: "/usr/local/tomcat/.OpenMRS/lucene/"
+            - name: {{ .Release.Name }}-data
+              subPath: "openmrs/activemq-data"
+              mountPath: "/usr/local/tomcat/.OpenMRS/activemq-data/"
+            - name: {{ .Release.Name }}-data
+              subPath: "openmrs/owa"
+              mountPath: "/usr/local/tomcat/.OpenMRS/owa/"
+            - name: {{ .Release.Name }}-data
```
Comment (rbuisson) [inline src/bahmni-helm/templates/apps/openmrs/openmrs-deployment.yml:98]: I don't think this is "data" per say. This is rather meant to deploy artifacts so fits into the distro volume instead. But right now our distros don't provide any OWAs so the path is absent. But we should keep the support for it.

File: src/bahmni-helm/templates/apps/openmrs/openmrs-deployment.yml [hunk 2]
```diff
@@ -86,7 +86,16 @@ spec:
             - name: {{ .Release.Name }}-data
               subPath: "openmrs/configuration_checksums"
               mountPath: "/usr/local/tomcat/.OpenMRS/configuration_checksums/"
-            - name: haiti-data
+            - name: {{ .Release.Name }}-data
+              subPath: "openmrs/lucene"
+              mountPath: "/usr/local/tomcat/.OpenMRS/lucene/"
```
Comment (rbuisson) [inline src/bahmni-helm/templates/apps/openmrs/openmrs-deployment.yml:91]: Would you know what's in this directory?

**Review by rbuisson — CHANGES_REQUESTED** [review]
Thanks! just few questions.

File: src/bahmni-helm/templates/apps/openmrs/openmrs-deployment.yml (hunk 2 above)
Comment (enyachoke) [reply src/bahmni-helm/templates/apps/openmrs/openmrs-deployment.yml:91]: This contains the lucene index files. If we don't put it in a volume we will have to reindex the db if the container is restarted.

**Pull request merged**